Production/python/Production.py

```diff
@@ -133,31 +134,38 @@
 if options.rerunTauReco:
     tau_collection = 'selectedPatTaus'
 
-    tauAtMiniTools.addTauReReco(process)
-    tauAtMiniTools.adaptTauToMiniAODReReco(process, options.reclusterJets)
+    tauAtMiniConfig = importlib.import_module('RecoTauTag.Configuration.tools.adaptToRunAtMiniAOD')
```

Comment:
why `importlib.import_module`? simple import doesn't work for some reason?

Reply:
It also works as `import RecoTauTag.Configuration.tools.adaptToRunAtMiniAOD as tauAtMiniConfig`, would it be more propriety way? I have seen that ID module is loaded this way and followed the same way.

Reply:
It is a legacy code. At some point we was using modified tauID customisation module in nonstandard location, so `import` didn't work. Currently, I think we should be able to get rid of `importlib` everywhere in `Production.py`.